yarn.lock

```diff
@@ -5,85 +5,103 @@
 "@comandeer/babel-plugin-banner@^1.0.0":
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/@comandeer/babel-plugin-banner/-/babel-plugin-banner-1.0.0.tgz#40bcce0bbee084b5b02545a33635d053c248356f"
+  integrity sha1-QLzOC77ghLWwJUWjNjXQU8JINW8=
```

Comment:
yarn v1.10 added an integrity field, so when cloning and running `yarn` locally, this field was added to each dependency. Just a heads up.